config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -9,6 +9,8 @@ en:
             failure: 'Could not create submission: %{error}'
           edit:
             attempt: :'course.assessment.assessments.assessment.attempt'
+          blocked:
+            message: 'This submission has been locked from viewing by the course staff.'
```

Comment:
I feel like the message here can be improved. Maybe something like:

```
Submissions for this assessment cannot be viewed once finalised.
```

The current message might be misinterpreted as this specific student's submission being locked by course staff. Not sure if I'm overthinking this - I'm open to suggestions.

Reply:
Updated